Automattic/wp-calypso
# State: Cleanup unused Reader action types

#### Changes proposed in this Pull Request

This PR removes some unused action types from the Reader state. Similar to #58393, but for the reader.

#### Testing instructions

Verify Calypso continues to build correctly and none of the removed action types are in use.
